src/homePage.js

```diff
@@ -0,0 +1,90 @@
+import 
+React , 
+{ Component 
+} from 'react';
+import {
+	Routes ,
+  Route ,
+  Link
+} from 'react-router-dom';
+//import { withRouter } from 'react-router'
+import Todolist from './todolist'
+import RegisterForm from "./registerPage";
+import './styles.css';
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+
+class App extends Component {
+  // React States 
+  constructor (props) {
+    super(props);
+      this.state = {
+        currentUser:{
+        email:'',
+        password:'',
+        taskList:[]
+        },
+        isloggedIn:false,
+      };
+  };
+
+  // User Authentication Function
+  handleSubmit = (event) => {
+
+    // Prevent page reload
+    event.preventDefault();
```

Comment:
what happen if we remove the event.preventDefault()

Reply:
Default actions to given component then cannot be prevented.